src/internal.c

```diff
@@ -18782,24 +18782,32 @@ int ProcessReplyEx(WOLFSSL* ssl, int allowSocketErr)
                                        &ssl->curRL, &ssl->curSize);
 
 #ifdef WOLFSSL_DTLS
-            if (ssl->options.dtls &&
-                    (ret == SEQUENCE_ERROR || ret == DTLS_CID_ERROR)) {
-                WOLFSSL_MSG("Silently dropping DTLS message");
-                ssl->options.processReply = doProcessInit;
-                ssl->buffers.inputBuffer.length = 0;
-                ssl->buffers.inputBuffer.idx = 0;
+            if (ssl->options.dtls) {
+                if ((ssl->options.handShakeDone && !IsEncryptionOn(ssl, 0))
```

Comment:
I don't understand how we can have a handshake done and not be encrypting.

What about when using an authentication only cipher suite (NULL cipher suite)?

Reply:
I think the reason we don't see an issue with `./configure --enable-dtls --enable-dtlscid --enable-dtls13 --enable-nullcipher` is because this is for DTLS only. Do we support DTLS with a NULL cipher? If so it does not appear we have a test case for it.

Reply:
`IsEncryptionOn` for dtls, only checks the epoch number of the received message. A null ciphersuite doesn't affect this logic.

Reply:
And `return ssl->keys.encryptionOn && (isSend ? ssl->encrypt.setup : ssl->decrypt.setup);` is true for a null cipher after the ccs message.